app/views/admin/adjustments/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= provide(:title, t('adjustment.title')) %>
+<div class="headline">
+  <h1><%= t('adjustment.title') %></h1>
+</div>
+
+<div class="col-md-3 col-sm-12">
+  <% if can? :manage_voting, User %>
+    <%= link_to t('vote_user.attendance_list'), attendance_list_admin_vote_users_path,
+                class: 'btn primary' %>
+  <% end %>
+  <% if can? :manage, Vote %>
+    <%= link_to t('vote.manage'), admin_votes_path, class: 'btn secondary' %>
+  <% end %>
+  <% if can? :manage, Agenda %>
+    <%= link_to t('agenda.manage'), admin_agendas_path, class: 'btn secondary' %>
+  <% end %>
+</div>
+
+<div class="col-md-9 col-sm-12">
+  <% if @agenda.present? %>
+    <%= @agenda %>
+  <% end %>
+
+  <br>
+
+  <%= simple_form_for(:vote_user, url: search_admin_vote_users_path, remote: true) do |f| %>
+    <%= f.input(:card_number, required: false, autofocus: true,
+                label: User.human_attribute_name(:card_number),
+                input_html: {data: {mask: '9999-9999-9999-9999'}}) %>
```

Comment:
Yeah, I fixed that - but do you think it is okay that you can show partial values?